src/Terrain/Environment.java

```diff
@@ -15,7 +15,21 @@ public class Environment implements Serializable
 
   public enum Weathers
   {
-    CLEAR, SNOW, RAIN, SANDSTORM
+    //      AW2 snow           AW2
+    CLEAR, SNOW(true, false), RAIN,
+    //  AWDS    AWDS snow
+    SANDSTORM, CHILL(true, false),
+    //   DoR snow        AW3/4 rain          DoR sandstorm
+    SLEET(true, false), SMOKE(false, true), SIROCCO;
```

Comment:
Is there any good way to tell what weather is in effect while in-game? I'm certainly not gonna memorize 8 different subtly-different overlay colors. Or maybe I haven't gotten to that yet.

Reply:
Also is RANDOM weather just going to cycle between everything? Or have game-themed subsets of weather? do you just not plan to support that?

Reply:
Uh, no, not really.
It's at least always tied to a global option or a CO activating it, at this point, which really limits the options.
Clearly, I need to add a little weather dial like Wargroove has, but it changes based on the tile you're mousing over.
(that is so not in scope for now, though)

Random is... yeah, I dunno. It's not really something I care about, and I really doubt the players will ask for it.
I would probably do separate randoms tied to each game, yeah.
It'd be fun to do a randomized local weather system, but that + overlays (like DoR COs) could make for some very unreadable gamestates (even if we made custom assets for every weather, which is _so_ not happening).